javaServices/coreJavaServices/src/joliex/util/TimeService.java

```diff
@@ -415,4 +422,38 @@ public Long getTimestampFromString( Value request )
 			throw new FaultException( "InvalidTimestamp", pe );
 		}
 	}
+
+	@RequestResponse
+	public String scheduleTimeout( Value request ) {
```

Comment:
Indentation is inconsistent with the rest of the code (I think I can send you a conf package if you're using Netbeans).